src/components/MenuItem/MenuItem.scss

```diff
@@ -0,0 +1,31 @@
+@import "./../../globalStyles/variables.scss";
+
+.MenuItem {
+    display: flex;
+    flex-wrap: wrap;
+
+    &__crossIcon {
```

Comment:
Also you need a `cursor: pointer` property on your icon. People need to see the hand icon when they put their mouse over the button to make it obvious that it's clickable